agent/app/agent_capability_unix.go

```diff
@@ -256,5 +261,16 @@ func checkFaultInjectionTooling() bool {
 			return false
 		}
 	}
+	return checkFaultInjectionModules()
+}
+
+// checkFaultInjectionModules checks for the required kernel modules such as sch_netem to be installed
+// and avaialble on the host before ecs.capability.fault-injection can be advertised
+func checkFaultInjectionModules() bool {
+	_, err := osExecWrapper.CommandContext(context.Background(), modInfoCmd, faultInjectionKernelModules).CombinedOutput()
```

Comment:
Should we have a timeout here? Maybe 60 seconds?

Reply:
sure, originally had 5 seconds but didn't want to include in case it affects agent start up time taking too long

Reply:
But without a timeout there is a risk that Agent gets stuck here for an indeterminate amount of time.

Reply:
yep agreed